holmes/plugins/toolsets/kafka.py

```
@@ -469,20 +498,106 @@ def _invoke(self, params: dict, context: ToolInvokeContext) -> StructuredToolRes
+                # This handles inactive groups
+                try:
+                    group_offsets_futures = client.list_consumer_group_offsets(
+                        [group_id], request_timeout=10
+                    )
+                    if group_offsets_futures.get(group_id):
+                        group_offsets = group_offsets_futures.get(group_id).result(timeout=15)
+                        # Build TopicPartition from the returned offsets
+                        for topic_partition in group_offsets.partitions:
+                            if topic_partition not in topic_partitions:
+                                topic_partitions.append(topic_partition)
+                except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)